includes/wikia/WikiaSQLIterator.class.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * WikiaSQLIterator
+ */
+
+class WikiaSQLIterator {
```

Comment:
Could be great if this class implemented an Iterator interface so we could supply it to a foreach loop instead of while. How does it sound?